lib/tasks_private/spec_helper.rake

```diff
@@ -65,42 +99,56 @@ class PopulateTower
       c.basic_auth(id, password)
     end
 
-    # Get API version.
+    @tower_data = tower_data
+
     uri = '/api/v1/config'
-    obj = JSON.parse(@conn.get(uri).body)
-    @version = Gem::Version.new(obj['version'])
+    config = get_obj(uri)
+    @version = Gem::Version.new(config['version'])
+    @tower_data.update { |td| td['config'] = config.slice('version') }
+
+    uri = '/api/v1/me'
+    me = get_obj(uri)
+    @tower_data.update { |td| td['user'] = me['results'].first.slice('id') }
   end
 
   def v3_2?
     @version >= Gem::Version.new('3.2')
   end
 
-  def create_obj(uri, data)
+  def create_obj(uri, data, dumped_fields = %w())
     del_obj(uri, data['name'])
     obj = JSON.parse(@conn.post(uri, data).body)
     puts "%s %s %s" % ["Created name=#{obj['name']}".ljust(40), "manager_ref/ems_ref= #{obj['id']}".ljust(30), "url=#{obj['url']}".ljust(10)]
+
+    dumped_data = obj.slice(*dumped_fields)
+    @tower_data.update { |td| td['items'][obj['name']] = dumped_data } unless dumped_data.empty?
```

Comment:
Can you move the `@tower_data.update` part out of this method?  Let it just does the creation of object in Tower.